### PR TITLE
Brianlmerritt kent use php 5.6 or 7

### PR DESCRIPTION
**Pull request by blm-unikent**
The composer.json changes should allow Moodle servers with php 5.5, 5.6 and 7.0 to run this plugin. Fix version issue in composer.json and added travis tests

### CODE REVIEW

**Comment by ryasmi** [comment]
Hi @brianlmerritt-kent, thank you for your contribution, I'm pretty happy with this, but could you please link me to some documentation for the version syntax you're using in `1.0.0 as 0.12.0`? I'm not sure what the exact behaviour is of the `as` in there.

**Comment by ryasmi** [comment]
Never mind I've found it https://getcomposer.org/doc/articles/aliases.md#require-inline-alias

**Comment by ryasmi** [comment]
What's the reason for your use of the `as`? I'm assuming it's so that the LearningLocker dependencies work, however, we also control the source of those dependencies so we should just update them to use 1.0.0.

**Comment by blm-unikent** [comment]
??Some learninglocker dependencies are asking for 0.12.0, so this avoids necessity of accepting less than stable



________________________________

**Comment by ryasmi** [comment]
The Learning Locker dependencies

- [Moodle Log Expander](https://github.com/LearningLocker/Moodle-Log-Expander)
- [Moodle to xAPI Translator](https://github.com/LearningLocker/Moodle-xAPI-Translator)
- [xAPI Recipe Emitter](https://github.com/LearningLocker/xAPI-Recipe-Emitter)

**Comment by blm-unikent** [comment]
Exactly!


https://github.com/LearningLocker/xAPI-Recipe-Emitter/blob/master/composer.json?



________________________________

**Comment by ryasmi** [comment]
Ok so I think we can apply the same changes from here in the Learning Locker dependencies.

- [composer.json in the Emitter](https://github.com/LearningLocker/xAPI-Recipe-Emitter/blob/master/composer.json#L15)
- [travis.yml in the Emitter](https://github.com/LearningLocker/xAPI-Recipe-Emitter/blob/master/.travis.yml#L2)
- [travis.yml in the Translator](https://github.com/LearningLocker/Moodle-xAPI-Translator/blob/master/.travis.yml#L2)
- [travis.yml in the Expander](https://github.com/LearningLocker/Moodle-Log-Expander/blob/master/.travis.yml#L2)

Obviously the composer.lock file will also need to change in the Emitter.

**Comment by ryasmi** [comment]
Once those changes have been made we can just use 1.0.0 and drop the use of `as` which is definitely preferred.

**Comment by blm-unikent** [comment]
?Yes, agreed

________________________________

**Comment by davidpesce** [comment]
Added necessary changes to dependencies (Emitter, Translator, and Expander)

- Translator - https://github.com/LearningLocker/Moodle-xAPI-Translator/pull/52
- Expander - https://github.com/LearningLocker/Moodle-Log-Expander/pull/44
- Emitter - https://github.com/LearningLocker/xAPI-Recipe-Emitter/pull/37

**Comment by davidpesce** [comment]
@ryansmith94 is this one approved for merge?

File: composer.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
         "learninglocker/statementfactory": "~0.0"
     },
     "require": {
-        "rusticisoftware/tincan": "0.40.0",
+        "php": "~5.5 || ^7.0.3",
+        "rusticisoftware/tincan": "1.0.0 as 0.12.0",
```
Comment (ryasmi) [inline composer.json]: Can we change this to just be 1.0.0 @davidpesce?

**Review by ryasmi — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (davidpesce) [reply composer.json]: Should be all set!

**Review by ryasmi — APPROVED** [review]

**Pull request merged**